include/seqan3/search/fm_index/concept.hpp

```diff
@@ -352,11 +339,9 @@ SEQAN3_CONCEPT BiFmIndexCursor = FmIndexCursor<t> && requires (t cur)
  * \{
  *
  * \typedef typename t::index_type index_type
- * \memberof seqan3::BiFmIndexCursor
  * \brief Type of the underlying SeqAn FM index (not the underlying SDSL index).
  *
  * \typedef typename t::size_type size_type
- * \memberof seqan3::BiFmIndexCursor
  * \brief Type for representing the size of the indexed text.
```

Comment:
the stuff still appears as part of the concept? Is it now not marked as a member anymore? If not, can you just do \memberof instead of relates?

Reply:
It's still part of the concept. The concept page looks the same as before.